jsapp/js/components/submissionDataTable.es6

```diff
@@ -266,14 +266,19 @@ class SubmissionDataTable extends React.Component {
   renderAttachment(type, filename) {
     const attachment = this.findAttachmentData(filename);
 
-    if (type === QUESTION_TYPES.image.id) {
-      return (
-        <a href={attachment.download_url} target='_blank'>
-          <img src={attachment.download_small_url}/>
-        </a>
-      );
+    if (attachment) {
+      if (type === QUESTION_TYPES.get('image').id) {
+        return (
+          <a href={attachment.download_url} target='_blank'>
+            <img src={attachment.download_small_url}/>
+          </a>
+        );
+      } else {
+        return (<a href={attachment.download_url} target='_blank'>{filename}</a>);
+      }
+    // In the case that an attachment is missing, don't crash the page
     } else {
-      return (<a href={attachment.download_url} target='_blank'>{filename}</a>);
+      return(t('Could not retrieve') + ' ' + filename);
```

Comment:
@duvld this is definitely a nitpick when applied to this error-handling code, but it's more important in other contexts: if you include a placeholder within the translatable string, and then replace that placeholder, our translators can do a better job, especially with right-to-left languages. For example:
```javascript
      return(t('Could not retrieve ##filename##').replace('##filename##', filename));
```

The convention used to be a triple underscore as the placeholder, but doing a quick search makes me think we evolved to `##thing##`. @magicznyleszek is that right? Also, is this PR okay to merge?

Reply:
Yes, we are using `##name##` as this makes it clear what we are looking for in the variable plus it is easier to have multiple variables in same string

Reply:
@duvld please make the change John is suggesting and then you can merge this :ship: :it: